### DISCUSSION TITLE
Não esotu conseguindo subir meu README

### DISCUSSION THREAD

**Discussion by LeonardoF27** [digitalinnovationone/dio-lab-open-source#42421] [Perguntas e Respostas] [answered]
![captura](https://github.com/user-attachments/assets/2b65c75f-4e56-4061-b5dc-233449e27c94)

**Comment by elidianaandrade** [comment, marked as answer]
Olá, @LeonardoF27! Esse problema ocorreu, pois você está alterando também o README na raiz do repositório, como pode observar em "changed files".

Para que o seu PR seja aceito, precisa enviar somente o seu Profile README, que seria esse arquivo com o mesmo nome de usuário (atente-se para maiúsculas e minúsculas, pois a validação é case-sensitive) na pasta community, ou seja, seu PR deve modificar apenas o arquivo community/LeonardoF27.md (verifique na aba "Files changed").
